### PR TITLE
Add missing params key

### PR DESCRIPTION
**Pull request by chris-rudmin**

### CODE REVIEW

File: content/content-management/multilingual.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ title = "Mon blog"
 weight = 2
 [Languages.fr.params]
 linkedin = "lien-francais"
-[Languages.fr.navigation]
```
Comment (bep) [inline content/content-management/multilingual.md]: I think the correct fix here is to remove the params above, i.e.  [Languages.fr.params] >  [Languages.fr]

**Comment by bep** [comment]
> if so, do you know why?

Not sure why you would need a "why", but this has been discussed in other places -- there is an open issue about it in the main repo. The short gist of it is that everything in a language section is put into `params` for that language.

**Comment by chris-rudmin** [comment]
@bep I must say very confusing. Is this the correct fix then?

**Comment by chris-rudmin** [comment]
IMO, if the ‘why’ is not clear, people might unknowingly run into namespace conflicts with params and localizations...

**Pull request merged**